common-util/web3.js

```diff
@@ -2,7 +2,7 @@ import Web3 from "web3";
 import olasAbi from "../data/ABIs/Olas.json";
 import tokenomicsAbi from "../data/ABIs/Tokenomics.json";
 
-const providerUrl = "https://ethereum.publicnode.com";
+const providerUrl = "https://ethereum-rpc.publicnode.com";
```

Comment:
just noticed this message [here](https://ethereum.publicnode.com/), so updated just in case 
<img width="1428" alt="image" src="https://github.com/user-attachments/assets/2bc32aed-7494-4b1d-aefc-6ca984e67528">